plyer/platforms/android/notification.py

```diff
@@ -41,8 +40,17 @@ class AndroidNotification(Notification):
     '''
 
     def __init__(self):
+        package_name = activity.getPackageName()
```

Comment:
So, the 'pm' and 'package_name' are only needed after
```python
if info.icon == 0:
```
So please can you move them to within that if? Otherwise there is no need to execute them?

Also, the 'self._channel_id' is assigned but never used?

Reply:
Hi, the if condition depends on `info` which depends on `pm.getActivityInfo` so I can't move the `pm` assignment any lower than it already is :)

And the `package_name` is used to assign `self._channel_id`. This logic is unchanged by this PR, I just created a temporary variable so that the result of `activity.getPackageName()` can be reused inside the if.  The channel id is used in code unrelated to this PR.

Reply:
Yeah, sorry, misread that :-). And have you been able to confirm that it work on a device?

Reply:
Yep, I have, and it does. :-)

This screenshot shows a notification generated by a version of plyer with the change.

![image](https://user-images.githubusercontent.com/194842/166641733-363e1a3d-acab-446b-a24e-5049e3d56dc3.png)

Reply:
Awesome! (I had to double check). Nice work. Thanks.